pkg/antctl/raw/multicluster/deploy/deploy_helper.go

```diff
@@ -91,6 +92,9 @@ func createResources(cmd *cobra.Command, apiGroupResources []*restmapper.APIGrou
 
 		obj, gvk, err := yaml.NewDecodingSerializer(unstructured.UnstructuredJSONScheme).Decode(rawObj.Raw, nil, nil)
 		if err != nil {
+			if string(content) == "Not Found" {
+				return errors.New("specified version tag is not found")
```

Comment:
```suggestion
				return errors.New("The specified version tag is not found")
```

Reply:
1. s/The/the/ (error shouldn't be captilized)
2. It could be more specific which resource is missing, so better to include the url:
```
if resp.StatusCode == 404 {
        return fmt.Errorf("manifest %s not found", manifest)
}
```